lib/modules/settingsNavigation.js

```diff
@@ -118,7 +118,7 @@ function onHashChange() {
 
 function onPopState(event) {
 	const state = typeof event.state === 'string' && event.state.split('/');
```

Comment:
This doesn't work anymore because URLs with `#res/settings` have more slashes than those with `#!settings`.

To fix this you could switch to `#res:settings` (like a pseudo-protocol), or change the parsing logic; I prefer the former, if you have no objections.